src/config/env/cloud.js

```diff
@@ -3,7 +3,7 @@ export default {
     db: 'mongodb://localhost/db',
     port: 3000,
     seedDB: false,
-    plugAndPlay: false,
+    plugAndPlay: true,
```

Comment:
!fix
lets keep that disabled for cloud -- which would be used by all developers -- which can lead to ddosing device by local servers
and enable that for heroku env